CONTRIBUTING.md

```diff
@@ -156,30 +121,30 @@ Make sure to cite any papers, algorithms or articles that can help
 understand the implementation of the feature. If you are implementing an
 algorithm described in a paper, add pointers to the section / steps.
 
-#### Code style
+Please use the [Google style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).
 
+### Testing
 
-Please review your changes for styling issues, clarity, according to the
-[PEP8 convention](https://www.python.org/dev/peps/pep-0008/). Correct
-any code style suggested by an analyzer on your changes.
-[PyCharm](https://www.jetbrains.com/help/pycharm/2016.1/code-inspection.html)
-has a code analyser integrated or you can use
-[pyflakes](https://github.com/PyCQA/pyflakes).
+Please add tests, especially with new code. Unit tests are located under
+`testing/`. They are straightforward to augment, but we understand it's the extra mile; it
+would still be appreciated if you provide something lighter (eg. in the
+commit messages or in the PR or issue text) that demonstrates that an
+issue was fixed, or a feature is functional.
 
-Do not address your functional changes in the same commits as any
-styling clean-up you may be doing on existing code.
+Consider that if you add test cases, they will ensure that your feature
+-- which you probably care about -- does not stop working in the
+future.
 
-#### Licensing
+### Licensing
 
 Ensure that you are the original author of your changes, and if that is
 not the case, ensure that the borrowed/adapted code is compatible with
-the MIT license.
+the [project's license](https://ivado-medical-imaging.readthedocs.io/en/latest/license.html).
```

Comment:
Could this be a relative link? Being absolute means this repo -- and all forks of it -- and that site are glued together.

Reply:
i would prefer it to be relative indeed, but i didn't find an easy way to redirect to the RTD's TOC entry. The fact that this file is a symlink (and in md format, which is further processed by Recommark for sphinx compatibility) makes it tricky...